.travis.yml

```diff
@@ -31,3 +31,4 @@ notifications:
   email:
     on_success: never
     on_failure: change
+  webhooks: https://hooks.slack.com/services/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
```

Comment:
Sorry, I didn't realise this was for Travis, I assumed it was going in the GitHub settings.

It needs to be encrypted, Slack doesn't restrict who can post to a webhook URL, AFAIK.

This URL only posts to #cli-firehose, so I'll leave it for now (I'm not expecting it to magically become a spam source, I'll delete it if it does). DM me on Slack when you're ready to change the URL in all of the wp-cli GitHub settings, and do an encrypted version of this PR.